### DISCUSSION TITLE
Welcome to py-protocol-layer Discussions!

### DISCUSSION THREAD

**Discussion by ONDC-Tech** [ONDC-Official/py-protocol-layer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by indu2598** [comment]
Hi all,

I want to run the python code but not able to get the idea, any help will be appreciated.

Thanking you in advance!
